profile/README.md

```diff
@@ -0,0 +1,46 @@
+![Virtual Coffee full logo - brackets and carets making a mug shape before the words "Virtual Coffee"](../virtual-coffee-full.png)
+
+
+:coffee: Welcome to Virtual Coffee! :coffee:
+======
+[Virtual Coffee](https://virtualcoffee.io) is a laid-back conversation with developers twice a week. It's the conversation that keeps going in slack. It's the online events that support developers at all stages of the journey. It's the place you go to make friends. Anyone can join! Whether you're thinking about getting into tech or have been in it for decades.
+
+Our Mission
+------
+Virtual Coffee's mission is to be a welcoming tech community that allows room for growth and mentorship at all levels, and to create meaningful opportunities for learning, leadership, and contribution for everyone.
+
+[Sponsor Us!](https://github.com/sponsors/Virtual-Coffee)
+------
+Here are some of the things that sponsorship money will go to support:
+
+* Twice-Weekly Coffee Chats
+* Member-led Presentations and Workshops
+* A weekly [podcast](https://virtualcoffee.io/podcast/) sharing the stories of our members
+* A monthly [newsletter](https://virtualcoffee.io/newsletter/) with Virtual Coffee news and interesting things our members are doing
+* Our growing Slack community
+* The Virtual Coffee Co-Working Room
+* Lightning Talks
+* The Virtual Coffee Hacktoberfest Initiative
+and lots more!
+Be sure to check out the [one-time](https://github.com/sponsors/Virtual-Coffee?frequency=one-time) sponsorship tiers for some more options!
+
+Learn with us!
+------
+[<img src="../TheJoyOfDevLifePreview.png" alt-text="A youtube video thumbnail showing 'Bekah Hawrot Wigel @BekahHW The Joy of Dev Life: A Bob Ross Guide' on a background of mountains and trees" width="400px">](https://www.youtube.com/c/virtualcoffeeio)
+
+FAQs :pencil:
+------
+### How do I become a member?
+To become a member of Virtual Coffee, all you need to do is attend [a Tuesday or Thursday Coffee](https://virtualcoffee.io/events/) and submit the form you'll receive at coffee.
+### What's the code of conduct?
+In order to create a welcoming and inclusive community, we pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+[Read more](https://github.com/Virtual-Coffee/virtualcoffee.io/blob/main/CODE_OF_CONDUCT.md) about how we enforce this and cultivate a community that assumes good intent.
```

Comment:
Let's change this link to https://virtualcoffee.io/code-of-conduct

Reply:
Ah, man. I had that first and second guessed myself.